drape_frontend/gui/skin.cpp

```diff
@@ -298,7 +298,7 @@ ReaderPtr<Reader> ResolveGuiSkinFile(string const & deviceType)
     }
   }
 
-  return reader;
+  return move(reader);
```

Comment:
No it should not. `reader` is an lvalue, unique_ptr have copy constructor deleted.
This is why it must be moved from result.